src/EfuncMsg.chpl

```diff
@@ -212,6 +212,29 @@ module EfuncMsg
                     }
                 }
             }
+            when (DType.UInt64) {
+                var e = toSymEntry(gEnt,uint);
+                select efunc
+                {
+                    when "popcount" {
```

Comment:
@reuster986 maybe these should mirror more of the `int64` ops like `parity`, also should we return an `int64` from these functions?

Reply:
I thought about it and I think the return values should be `int64` or `int` in Chapel.

Reply:
I am thinking we need `cumsum` and `parity`.
`cumsum` returns `uint64` or `uint` in Chapel
`parity` returns `uint64` or `uint` in Chapel

Reply:
Alright, I have changed them to be of the form `a.a = clz(e.a):int;`.

Do you think that approach is what we would want or instead something like: `a.a = clz(e.a:int);`, where the input is cast to an `int` rather than the result. I believe that there could be different behavior in some cases between the two.

Reply:
I don't have a strong opinion either way, but I'm curious why an `int64` return is preferable, since all results are guaranteed to be non-negative anyway?

Reply:
> I don't have a strong opinion either way, but I'm curious why an `int64` return is preferable, since all results are guaranteed to be non-negative anyway?

ok, i don't have a comeback for that ;-) except that it's the same as the return for `int64` but maybe that doesn't matter.
I hate to whip @bmcdonald3 around but I am ok with changing them back to `uint`.

Reply:
thanks! no big deal either way on the return type, but this way seems consistent with what chapel already does.